helix-term/src/ui/prompt.rs

```diff
@@ -19,7 +20,8 @@ use helix_view::{
 };
 
 type PromptCharHandler = Box<dyn Fn(&mut Prompt, char, &Context)>;
-pub type Completion = (RangeFrom<usize>, Cow<'static, str>);
+
+pub type Completion = (RangeFrom<usize>, Cow<'static, str>, Option<Style>);
```

Comment:
Rather than passing a style as a separate enum member, this type should replace the `Cow<'static, str>` with a `Span<'static>` from the tui crate. It's effectively the same thing:

```rust
pub struct Span<'a> {
    pub content: Cow<'a, str>,
    pub style: Style,
}
```

Reply:
Ok, I'm using `Span` instead of adding a 3rd tuple member now